### PR TITLE
feat: tests for alert channels settings

### PR DESCRIPTION
**Pull request by ahmadshaheer**
#### Related Issues / PR's
close https://github.com/SigNoz/signoz/issues/5562

#### Screenshots

NA

### CODE REVIEW

File: frontend/src/mocks-server/__mockdata__/alerts.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+export const allAlertChannels = [
+	{
```
Comment (SagarRajput-7) [inline frontend/src/mocks-server/__mockdata__/alerts.ts:2]: In below responses, just make sure once that you are not exposing any companies info, and if in doubt replace it with random and test fillers

**Comment by YounixM** [comment]
@ahmadshaheer : PR looks good. Please address @SagarRajput-7 's comment and we can merge this.

File: frontend/src/mocks-server/__mockdata__/alerts.ts (hunk 1 above)
Comment (ahmadshaheer) [reply frontend/src/mocks-server/__mockdata__/alerts.ts:2]: Done :+1: . makes sense :slightly_smiling_face:

**Comment by ahmadshaheer** [comment]
> @ahmadshaheer : PR looks good. Please address @SagarRajput-7 's comment and we can merge this.

Done :+1: , @YounixM. Thanks :pray: :raised_hands: :slightly_smiling_face:

**Review by YounixM — APPROVED** [review]

**Review by vikrantgupta25 — APPROVED** [review]

**Pull request merged**